src/components/Home/Map/Search.js

```diff
@@ -109,17 +109,12 @@ export default function Search({ zoomOnCluster }) {
         disabledDate={disableFutureDates}
         onCalendarChange={handleCalendarChange}
       />
-      <div>
-        <label>
-          {/* Revisit time values -one week instead? */}
-          Quick Search
-          <select onChange={onChange} value={quickSelect} name="quicksearch">
-            <option value="">- Search Recent Reports -</option>
-            <option value="30">Within Past 30 Days</option>
-            <option value="10">Within Past 10 Days</option>
-          </select>
-        </label>
-      </div>
+
+      <Select onChange={onChange} value={quickSelect} name="quicksearch">
+        <Option value="">- Search Recent Reports -</Option>
+        <Option value="30">Within Past 30 Days</Option>
+        <Option value="10">Within Past 10 Days</Option>
```

Comment:
I like this feature with the custom hook, you confirmed with me on your local machine.